packages/core/src/Engine.ts

```diff
@@ -307,7 +331,12 @@ export class Engine extends EventDispatcher {
     if (!this._isPaused) return;
     this._isPaused = false;
     this.time._reset();
-    this._requestId = requestAnimationFrame(this._animate);
+    const { xrManager } = this;
+    if (xrManager) {
+      this._requestId = xrManager.sessionManager.requestAnimationFrame(this._animate);
+    } else {
+      this._requestId = requestAnimationFrame(this._animate);
+    }
   }
```

Comment:
- I found this function has bug before, not consider not vSyncMode(setTimeOut)
- I want deprecate `resume`,rename `pause` to `stop`,only `run ` and `stop` is enough

Reply:
Done.